src/relay/backend/graph_executor_codegen.cc

```diff
@@ -551,10 +713,14 @@ class GraphExecutorCodegen : public backend::MemoizedExprTranslator<std::vector<
   Map<Expr, Array<IntegerArray>> storage_device_map_;
   /*! \brief lowered funcs */
   std::unordered_map<std::string, IRModule> lowered_funcs_;
+  /*! \brief lowered funcs */
+  Map<String, FunctionInfo> function_metadata_;
   /*! \brief name map */
   std::unordered_map<std::string, size_t> name_map_;
   /*! \brief compile engine */
   CompileEngine compile_engine_;
+  /*! \brief main function name */
+  const String kMainFuncStr = "main_func";
```

Comment:
i wonder if this should be `mod_name`, as passed to GraphExecutorFactory?

Reply:
My perception is that all the functions are in the namespace of mod_name, thus my opinion it should be the main function of <mod_name>.

Reply:
I've had a discussion with @giuseros . So the AoT currently uses _tvm_<mod_name>_run_func for the main function.
Few options, we can consider :

A1 : run_func
A2 : main_func
A3 : _tvm_<mod_name>_run_func ( + all operators function names as well for consistency)
A4 : _tvm_<mod_name>_main_func ( + all operators function names as well for consistency)

Reply:
i sort of favor A3 (though not sure we need the "_func" part), since this function essentially replaced GraphExecutor#run.

Reply:
Cool! I like _tvm_<mod_name>_run. 

@giuseros any objection?

Reply:
So there is a subsequent PR @giuseros  that will introduce the mangling of names to include \_tvm\_<mod_name>_ prefix, therefore I will leave this at just "run" for now.